l10n-pocket/en/pocket/feature-pages.ftl

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+## Save to Pocket URL: https://dev.tekcopteg.com/save-to-pocket/
+## Get Inspired URL: https://dev.tekcopteg.com/get-inspired/
+## Pocket and Firefox URL: https://dev.tekcopteg.com/get-inspired/
```

Comment:
Since some strings are reused within those 3 files, it would make better sense to keep the URLs at the top of the file. I'll put the triple hash here, then. Thanks for the look out!